pkg/proxy/envoyproxy.go

```diff
@@ -86,3 +95,59 @@ func (r *EnvoyRedirect) Close() {
 		envoyProxy.RemoveListener(r.id)
 	}
 }
+
+// Log does access logging for Envoy
+func (r *EnvoyRedirect) Log(pblog *envoy.HttpLogEntry) {
+	log.Infof("%s: Access log message: %s", pblog.CiliumResourceName, pblog.String())
+
+	var scheme, path, method string
+	headers := make(http.Header)
+	for _, header := range pblog.Headers {
+		switch header.Key {
+		case ":path":
+			path = header.Value
+		case ":method":
+			method = header.Value
+		case "x-forwarded-proto":
+			scheme = header.Value
+		case ":authority": // Envoy uses HTTP/2 representation also for HTTP/1.1
+			headers.Add("host", header.Value)
+		default:
+			headers.Add(header.Key, header.Value)
+		}
+	}
+
+	url := url.URL{
```

Comment:
Try avoiding variable names that have package names

Reply:
fixed. Also fixed the Oxy -references in comments you mentioned above.